app/modules/fcm/controllers/cytoscape.js

```diff
@@ -309,6 +310,16 @@ angular.module('pcApp.fcm.controllers.cytoscapes', [])
             return input;
         };
 
+        $scope.getUserId = function () {
+            var userId = '';
+            if ($scope.user != null || $scope.user != '') {
+                var userPathArray = $scope.user.state.userPath.split('/');
+                if (userPathArray.length > 2)
+                    userId = parseInt(userPathArray[userPathArray.length - 2], 10);
+            }
+            return userId;
+        };
+
```

Comment:
@mohammedbahja appart from what @FabiApfelkern said, you shouldn't store the userID like that.
First of all, you should keep the full user address (for example: "https://adhocracy-frontend-stage.policycompass.eu/api/principals/users/0000003/")
Also all request from authenticated users have two extra Headers. X-User-Path which contains the full url path of the user and the X-User-Token which you should always check against Adhocracy.
![image](https://cloud.githubusercontent.com/assets/1640100/14250177/866f1c98-fa7d-11e5-8901-21ebb02f743a.png)